pkgs/top-level/all-packages.nix

```diff
@@ -10330,7 +10330,9 @@ in
 
   cmake-format = python3Packages.callPackage ../development/tools/cmake-format { };
 
-  cmake-language-server = python3Packages.callPackage ../development/tools/cmake-language-server { };
+  cmake-language-server = python3Packages.callPackage ../development/tools/cmake-language-server {
+    inherit cmake;
```

Comment:
Explicitly specify `cmake`. If the cmake python module was packaged in the future it would break the check phase.

Reply:
you want
```suggestion
    inherit (pkgs) cmake;
```

your code would still pull from self, which is pythonPackages in this case

Reply:
:+1: Will fix